components/clarity-repl/src/repl/session.rs

```diff
@@ -823,14 +835,41 @@ impl Session {
             }
         };
 
-        let new_height = self.advance_chain_tip(count);
+        let new_height = self.advance_stacks_chaintip(count);
         format!("{} blocks simulated, new height: {}", count, new_height)
             .green()
             .to_string()
     }
+    fn parse_and_advance_burn_chaintip(&mut self, command: &str) -> String {
+        let args: Vec<_> = command.split(' ').collect();
 
+        if args.len() != 2 {
+            return format!("{}", "Usage: ::advance_burn_chaintip <count>".red());
+        }
+
+        let count = match args[1].parse::<u32>() {
+            Ok(count) => count,
+            _ => {
+                return format!("{}", "Unable to parse count".red());
+            }
+        };
+
+        let new_height = self.advance_burn_chaintip(count);
+        format!("{} blocks simulated, new height: {}", count, new_height)
+            .green()
+            .to_string()
+    }
+
+    pub fn advance_stacks_chaintip(&mut self, count: u32) -> u32 {
+        self.interpreter.advance_stacks_chaintip(count)
+    }
+    pub fn advance_burn_chaintip(&mut self, count: u32) -> u32 {
+        self.interpreter.advance_stacks_chaintip(count);
+        self.interpreter.advance_burn_chaintip(count)
+    }
+    // Not exposed/documented but still supported for convenience
     pub fn advance_chain_tip(&mut self, count: u32) -> u32 {
-        self.interpreter.advance_chain_tip(count)
+        self.interpreter.advance_stacks_chaintip(count)
```

Comment:
@tippenein This should call `advance_burn_chaintip`to keep the same behavior as before